physionet-django/console/templates/console/credential_processing.html

```diff
@@ -22,10 +22,10 @@
         <a class="nav-link" id="reference-tab" data-toggle="tab" href="#reference" role="tab" aria-controls="reference" aria-selected="false">Reference Check {{ reference_applications|task_count_badge|safe }}</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link" id="response-tab" data-toggle="tab" href="#response" role="tab" aria-controls="response" aria-selected="false">Reference Response {{ response_applications|task_count_badge|safe }}</a>
+        <a class="nav-link" id="response-tab" data-toggle="tab" href="#response" role="tab" aria-controls="response" aria-selected="false">Awaiting Reference Response {{ response_applications|task_count_badge|safe }}</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link" id="final-tab" data-toggle="tab" href="#final" role="tab" aria-controls="final" aria-selected="false">Final Review {{ final_applications|task_count_badge|safe }}</a>
+        <a class="nav-link" id="final-tab" data-toggle="tab" href="#final" role="tab" aria-controls="final" aria-selected="false">Final Reference Review {{ final_applications|task_count_badge|safe }}</a>
```

Comment:
This is the final review of the application, not just the reference check. I think leaving this as "Final review" may be better?

Reply:
I think after approving the previous tab checks e.g ID (identity of applicant), the 'Final Review' tab has the addition of the response from the reference before full approval, therefore adding 'Final Reference Review'  adds clarity on what additions to look for in the checking this tab.